### PR TITLE
Add validator-manager module

### PR DESCRIPTION
**Pull request by cowboy-bebug**
This PR adds `prml-validator-manager` runtime module.

### CODE REVIEW

**Review by hoani — APPROVED** [review]
LGTM - local integration tests also passing

File: prml/validator-manager/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+// Plug is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Plug. If not, see <http://www.gnu.org/licenses/>.
+
+//! # Proof of Authority module.
```
Comment (jordy25519) [inline prml/validator-manager/src/lib.rs]: Could expand this abit more
things like
- intended for use with aura and grandpa
- implements session handler trait
- sudo controls the validator set via extrinsics

File: prml/validator-manager/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Plug. If not, see <http://www.gnu.org/licenses/>.
+
+//! # Validator Manager module.
+//!
+//! This module provides configurable proof-of-authority through adding and
+//! removing of validators, controlled by root. It is intended to be used in
+//! conjunction with AURA and GRANDPA.
```
Comment (jordy25519) [inline prml/validator-manager/src/lib.rs:21]: 🥇

**Review by jordy25519 — APPROVED** [review]
🚀

**Comment by jordy25519** [comment]
Needs squash and sign

**Pull request merged**